Sources/Quick/Configuration/QuickConfiguration.swift

```diff
@@ -26,9 +26,9 @@ internal func qck_enumerateSubclasses<T: AnyObject>(_ klass: T.Type, block: (T.T
     for i in 0..<classesCount {
         subclass = classes[Int(i)]
 
-        if let superclass = class_getSuperclass(subclass), superclass === klass {
-            block(subclass as! T.Type) // swiftlint:disable:this force_cast
-        }
+        guard let superclass = class_getSuperclass(subclass), superclass == klass else { continue }
```

Comment:
I'll need some help here. When using `===` there, there is still a runtime crash (EXC_BAD_ACCESS). For now I did `==` and it kinda works, but we probably need to find a real problem? 🤔

Reply:
Looks like using `==` is valid here:

https://github.com/apple/swift/blob/115a3627c5fd413d2bf7fb92b275559f0f82c5a7/stdlib/public/core/Builtin.swift#L147-L162